packages/virtualdom/src/index.ts

```diff
@@ -175,7 +175,8 @@ export type ARIAAttrNames =
   | 'aria-valuemin'
   | 'aria-valuenow'
   | 'aria-valuetext'
-  | 'role';
+  | 'role'
+  | 'tabIndex';
```

Comment:
`tabindex` (not no capitalisation) is already defined in `ElementAttrNames` 

https://github.com/jupyterlab/lumino/blob/579425109f937b7e1842eb9a6efacbc585dfd0e3/packages/virtualdom/src/index.ts#L112-L114

I wonder if it would be better to instead modify `createTabARIA` signature so that it returns `ElementARIAAttrs & ElementAttrNames` (and keep the capitalisation the same).

Reply:
Hi, the requested changes have just been made in recent commit.